provider/gcp/sia-gce/authn.go

```diff
@@ -36,24 +34,22 @@ func GetGCEConfig(configFile, profileConfigFile, metaEndpoint, region string, pr
 			log.Printf("Unable to process environment settings: %v\n", err)
 			// if we do not have settings in our environment, we're going
 			// to use fallback to retrieve values from the context ( metadata etc )
-			config, profileConfig, err = options.InitGenericProfileConfig(metaEndpoint, "", "", provider)
+			config, _, err = options.InitGenericProfileConfig(metaEndpoint, "", "", provider)
```

Comment:
does this not change the behavior of the existing sia? We're ignoring the profile value we get from this API and instead get it from the GetGCEAccessProfile which seems to give preference to the config file as opposed to metadata.